source/_integrations/icloud.markdown

```diff
@@ -3,12 +3,27 @@ title: "iCloud"
 description: "Instructions on how to use iCloud to track devices in Home Assistant."
 logo: icloud.png
 ha_category:
+  - Hub
```

Comment:
This is not a Hub. Please remove this category.

Reply:
Fixed, I thought "hub of services", but it's actually a real hub device 😄